webapp/src/Entity/Problem.php

```diff
@@ -108,12 +101,15 @@ class Problem extends BaseApiEntity implements
     #[Serializer\Exclude]
     private ?string $problemstatement_type = null;
 
-    #[ORM\Column(options: [
-        'comment' => 'Whether this problem is a multi-pass problem.',
-        'default' => 0,
-    ])]
+    public const TYPE_PASS_FAIL = 1;
```

Comment:
Should we use an int backed enum for this?

Reply:
It can have multiple types at once, can I do this with a standard enum?

Reply:
Ah fair point. There is maybe a enum bitset package but I can't find it. Let’s stick to constants for now